.github/workflows/release.yml

```diff
@@ -84,13 +84,29 @@ jobs:
         run: |
           cd ${{env.BIN_DIR}}
           sha256sum ${{env.RELEASE_BIN_FILENAME}} > ${{env.RELEASE_BIN_FILENAME}}.sha256
-      - name: Upload Artifact
+      - name: Import GPG key
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@v5
```

Comment:
Since this touches our private key, we should likely pin it to the exact version.